src/events/pollVotationResults.js

```diff
@@ -10,11 +10,15 @@ module.exports = {
       return;
     }
 
+    if (!embeds[0]?.fields) {
+      return;
+    }
+
     const userMentionField = embeds[0]?.fields.find(
       (field) => field.name === 'poll_question_text'
     );
```

Comment:
please cover the case where this is undefined
```suggestion
    const userMentionField = embeds[0].fields.find(
      (field) => field.name === 'poll_question_text'
    );
    
    if (userMentionField === undefined) return;
```

Reply:
Completed